contracts/test/TestPaymasterConfigurableMisbehavior.sol

```diff
@@ -44,6 +45,9 @@ contract TestPaymasterConfigurableMisbehavior is TestPaymasterEverythingAccepted
     function setExpensiveGasLimits(bool val) public {
         expensiveGasLimits = val;
     }
+    function setPreGasHigherThanAcceptance(bool val) public {
```

Comment:
you have `setGasLimits` for this, setting all 3 gas params

Reply:
fixed